app/views/hub/clients/show.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= content_for :user_return_link_name, t(".return_to_clients") %>
+<%= content_for :user_return_link_name, t("hub.clients.navigation.return_to_clients") %>
 <%= content_for :user_return_link_path, hub_clients_path %>
```

Comment:
The repetition of these two lines across the different views seems like a clear cut case for including this in either a shared layout or in a shared partial. If refactored into a shared template, I would update the i18n key to use ".return_to_clients" and put the key whereever it needed to be based on the location of the layout/partial.